python/py-last/Portfile

```diff
@@ -20,17 +20,18 @@ homepage            https://github.com/pylast/pylast
 
 maintainers         {@catap korins.ky:kirill} openmaintainer
 
-checksums           rmd160  9ee82f7ae02ddc590b586130ea017d8e4dc70207 \
-                    sha256  5018b66c2b4632d71abd8103b73e66e4dd14c6609a8f7ba7e5aa26c736df2df8 \
-                    size    43871
+checksums           rmd160  e85c802f97f97b09b7ed74685b5df7817732864e \
+                    sha256  89300fdcdf423d7be0606bdc44da27c3b48c4d73aa1d4cb12672cc006c979bdc \
+                    size    40517
 
+python.pep517       yes
 python.versions     37 38 39 310
 
 if {${name} ne ${subport}} {
     depends_build-append \
-                    port:py${python.version}-setuptools \
-                    port:py${python.version}-setuptools_scm \
-                    port:py${python.version}-setuptools_scm_git_archive
+                    port:py${python.version}-hatch-vcs \
+                    port:py${python.version}-hatchling \
+                    port:py${python.version}-setuptools
```

Comment:
it looks like you could/should specify the correct `python.pep517_backend` instead of adding the `hatch` dependencies manually

Reply:
Thanks.